iohk-monitoring/src/Cardano/BM/Backend/Switchboard.lhs

```diff
@@ -124,8 +135,14 @@ instance IsEffectuator Switchboard a where
                     else atomically $ TBQ.writeTBQueue q i
 
         sb <- readMVar (getSB switchboard)
-        writequeue (sbQueue sb) item
 
+        -- First we check to see if we have the switchboard running.
+        -- If the switchboard is running, we write to the queue,
+        -- otherwise we report the error on the stderr.
```

Comment:
the code is self-explanatory

Reply:
A lot of things are self-explanatory until you need to figure out what they do. A couple of lines of comments go a long way.